app/templates/check_results.html

```diff
@@ -259,6 +259,55 @@ <h3>
                             </table>
                         </div>
                     </div>
+
+                    <div class="card p-1 bg-light border rounded-3 mb-3">
+                        {{ card_header("TLS: Mail Exchange", result.domain.tls.mail_transfer) }}
+                        <div class="card-body">
+                            {% for mail_exchange in result.domain.tls.mail_transfer.mail_exchanges %}
+                            <table class="table">
+                                <tr>
+                                    <td class="label">{% trans %}Host{% endtrans %}</td>
+                                    <td><code>{{ mail_exchange.host }}</code></td>
+                                </tr>
+                                <tr>
+                                    <td class="label">{% trans %}Port{% endtrans %}</td>
+                                    <td><code>{{ mail_exchange.port }}</code></td>
+                                </tr>
+                                <tr>
+                                    <td class="label">{% trans %}Connected{% endtrans %}</td>
+                                    <td>{% if mail_exchange.connected %}{% trans %}yes{% endtrans %}{% else %}{% trans %}no{% endtrans %}{% endif %}</td>
+                                </tr>
+                                {% if mail_exchange.is_certificate_valid is not none %}
+                                <tr>
+                                    <td class="label">{% trans %}Certificate is valid{% endtrans %}</td>
```

Comment:
the field is empty in polish translation:
![image](https://github.com/CERT-Polska/mailgoose/assets/1233067/72dfae60-9fdc-4efb-ab2f-0239ef3eb6dd)